app/views/images/_index.html.erb

```diff
@@ -3,8 +3,9 @@
   <p id="no-images">No images uploaded. </p>
   <%end%>
 <table class="warpables-all table table-striped">
-  <tbody>
-    <% @map.warpables.each do |warpable| %>
+  <tbody data-url = "<%= sort_images_path %>">
+    <% ary = @map.warpables.sort { |a, b| a.position <=> b.position } %>
```

Comment:
```
    <% ary = @map.warpables.sort { |a, b| a.position <=> b.position } %>
```
@gauravano 
This ```position``` is not being recognized even though it is added into the table.
After adding the column position from the rails terminal and migrating it did resolve the problem for me.